docs/how-ubuntu-is-made/concepts/version-strings.md

```diff
@@ -436,6 +444,7 @@ List of this and further related examples:
 
 |       | Old            | New                    |
 | ----- | -------------- | ---------------------- |
+| devel | `3.0-1ubuntu1` | `3.1-1ubuntu2`         |
```

Comment:
nope - this whole section is backport from ubuntu-devel - ubuntu devel itself does not belong here